src/fairchem/core/trainers/base_trainer.py

```diff
@@ -571,6 +575,9 @@ def load_checkpoint(
         self.step = checkpoint.get("step", 0)
         self.best_val_metric = checkpoint.get("best_val_metric", None)
         self.primary_metric = checkpoint.get("primary_metric", None)
+        self.config["cmd"]["parent"] = checkpoint["config"]["cmd"].get(
```

Comment:
where is this parent actually used?

Reply:
I dont think its read anywhere, i believe it was intended to link fine tuning runs to their parent runs. @mshuaibii is this right?

Reply:
lets remove if its not used

Reply:
Yeah it was introduced to organize things in wandb by this entry. But you guys probably have a different solution at this point so ommitting is fine